src/controller/java/generated/java/matter/controller/cluster/clusters/ActionsCluster.kt

```diff
@@ -168,8 +164,7 @@ class ActionsCluster(private val controller: MatterController, private val endpo
 
   suspend fun stopAction(actionID: UShort, invokeID: UInt?, timedInvokeTimeoutMs: Int? = null) {
     val commandId: UInt = 4u
-    val timeoutMs: Duration =
-      timedInvokeTimeoutMs?.let { Duration.ofMillis(it.toLong()) } ?: Duration.ZERO
+    val timeoutMs: Duration? = timedInvokeTimeoutMs?.let { Duration.ofMillis(it.toLong()) }
```

Comment:
How about passing in a Duration as an input to begin with instead of a variable suffixed with "ms" ?

Reply:
This should read like:

```kotlin
suspend fun stopAction(actionID: UShort, invokeID: UInt?, timedInvokeTimeout: Duration? = null) {
  val commandId: UInt = 4u; 
  // ...
 val request: InvokeRequest =
      InvokeRequest(
        CommandPath(endpointId, clusterId = CLUSTER_ID, commandId),
        tlvPayload = tlvWriter.getEncoded(),
        timedRequest = timedInvokeTimeout
      )
  //...
}
```

can the type of commandId be inferred and we just have `val commandId = 4u` ?

Reply:
But the type of timedInvokeTimeout is not defined as type of Duration in IDL file, we need to do extra magic to translate to Duration specifically for field timedInvokeTimeout, it is doable, but not worthwhile.

Reply:
the value of commandId is defined in IDL file, we can not infer it.